Source/UIViewController+STZPopupView.swift

```diff
@@ -136,7 +136,7 @@ extension UIViewController {
         dismissAnimation()
     }
 
-    private func completionDismissAnimation(finished: Bool) {
+    fileprivate func completionDismissAnimation(_ finished: Bool) {
```

Comment:
I prefer not to use underscore.
`private func completionDismissAnimation(finished: Bool)`